ksy_schema.json

```
@@ -99,16 +99,23 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
-        "id": { "$ref": "#/definitions/Identifier" },
-        "title": { "type": "string" },
+        "id": {
+          "$ref": "#/definitions/Identifier",
+          "description": "unique string that identifies this format. It is recommended to use the same name, as basename of the file with the format description. This name also will be used to infer name of type when generate parser\n\nThis field is mandatory on top-level definition and shouldn't be used at nested levels"
+        },
+        "title": {
+          "type": "string",
+          "description": "brief name of this format, for example 'Windows PE executable'"
+        },
         "application": {
           "anyOf": [
             { "type": "string" },
             {
               "type": "array",
               "items": { "type": "string" }
             }
-          ]
+          ],
+          "description": "name(s) of application(s) in free form that uses this format"
```

For clarity I would write this as

```suggestion
          "description": "name(s) of application(s) that uses this format, in free form"
```